templates/base.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Taxi Service</title>
+    <meta charset="UTF-8">
```

Comment:
need to create block `title`
![image](https://github.com/user-attachments/assets/29db2fb8-d64a-47e6-89c8-017cd6031df6)